boards/arduino/nicla_vision/arduino_nicla_vision_stm32h747xx_m7.dts

```diff
@@ -192,3 +186,81 @@ zephyr_udc0: &usbotg_hs {
 	phys = <&otghs_ulpi_phy>;
 	status = "okay";
 };
+
+&i2c3 {
+	status = "okay";
+	pinctrl-0 = <&i2c3_scl_pa8 &i2c3_sda_pc9>;
+	pinctrl-names = "default";
+	clock-frequency = <I2C_BITRATE_FAST>;
+
+	gc2145: gc2145@3c {
+		compatible = "galaxycore,gc2145";
+		reg = <0x3c>;
+		status = "okay";
+
+		port {
+			gc2145_ep_out: endpoint {
+				remote-endpoint = <&dcmi_ep_in>;
+			};
```

Comment:
FYI only, there is currently a bug somewhere in the python wrappers around the devicetree to support circular phandle references.

- https://github.com/zephyrproject-rtos/zephyr/issues/57708
- https://github.com/zephyrproject-rtos/zephyr/pull/74415 (workaround)

This does not affect this PR: the workaround is not merged yet.